scripts/settings/default/main.lua

```diff
@@ -83,6 +83,7 @@ xi.settings =
     GIL_RATE        = 1.000, -- Multiplies gil earned from quests.  Won't always display in game.
     BAYLD_RATE      = 1.000, -- Multiples bayld earned from quests.
     EXP_RATE        = 1.000, -- Multiplies exp earned from fov and quests.
+    FOV_EXP_RATE	= 1.000, -- Multiplies exp from FoV.
```

Comment:
"not using" looks more like "ceased to exist" but IDK when that happened. Worth mentioning (maybe add to the comment) that the conf exp setting ALSO affects the exp number. I've been actually reducing the FoV exp to compensate, because I don't want to have everyone farm EP while living off pages.

Reply:
oh..look at the line directly above this @Lynnea1320 `Multiplies exp earned from fov and quests.` just needed to you know..actually use it in teh global.

Reply:
well. they can be split. I can see wanting to not combine them. but update the comment!

Reply:
Ah yeah because addExp uses the exp rate. Doesn't that mean quests are using that too then? Maybe the addExp function needs to be overhauled to be a little more selective

Reply:
> Ah yeah because addExp uses the exp rate. Doesn't that mean quests are using that too then? Maybe the addExp function needs to be overhauled to be a little more selective

no can't really separate them in a sane manner, just have to remember that if you double exp in conf, your exp is already double before hitting this in lua. so then 2x here becomes 4x. This is why my fov exp is half, since my conf already boosts it. I looked at splitting them before its not worth the attempt.

Reply:
> oh..look at the line directly above this @Lynnea1320 `Multiplies exp earned from fov and quests.` just needed to you know..actually use it in the global.
> 
> well. they can be split. I can see wanting to not combine them. but update the comment!

As you can clearly see in regimes.lua, it does not multiply the exp earned from FoV lol